scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -328,6 +329,14 @@ function MetaTagsDirective(api) {
                 scope.adding = false;
             };
 
+            scope.$watch('adding', function() {
+                if (scope.adding) {
+                    $timeout(function() {
```

Comment:
this will trigger full digest which is not needed for elem focus, better add `0, false` arguments